map-demo/app/views/map.scala.html

```diff
@@ -13,11 +13,54 @@
                 <span>This is a new service – your <a href="#">feedback</a> will help us to improve it.</span>
             </p>
         </div>
+        
+        <div class="error-summary" id="error" role="group" aria-labelledby="error-summary-heading-example-1" tabindex="-1" hidden="true">
+            <h1 class="heading-medium error-summary-heading" id="postcode-error">
+            </h1>
+            <p>
+                Make sure your search is spelled correctly. Try adding a city, county or post code
+            </p>
+        </div>
+            <h4 class="heading-small">Layer controls</h4>
+            <div>
+                <form>
```

Comment:
This form and heading for layer controls won't be needed, was removed from Ash's branch